### PR TITLE
added interval that refreshes the page every minute

### PR DESCRIPTION
**Pull request by jorge8989**

### CODE REVIEW

**Comment by jorge8989** [comment]
@messutied

File: src/js/app.js [hunk 1]
```diff
@@ -4,3 +4,7 @@ window.setLocale = function (el) {
     console.log(el.value);
     window.location.href = window.location.href.split("?")[0] + "?lang=" + el.value;
 }
+
+var autoRefreshInterval = setInterval(function() {
```
Comment (messutied) [inline src/js/app.js]: No hace falta definir una variable ^^

Comment (jorge8989) [reply src/js/app.js]: Oops

**Comment by jorge8989** [comment]
Captura de Pantalla 2019-07-12 a la(s) 23 20 48
@messutied I tried placing the message in a couple of places. This is the best design I came up with.
This looks fine if you ask me.

I saw other status pages from different platforms, but none of them had a message about auto-refresh. I only saw messages like "updated a few seconds ago" or "updated 10 secs ago.", I guess referring to the status, not the browser.

**Comment by messutied** [comment]
And those other pages once they auto-refresh the message goes back to "updated 0 secs ago"?

**Comment by jorge8989** [comment]
I didn't see these pages refreshing. I waited for some time inspecting the pages with dev tools, and they didn't update/refresh automatically. I didn't see the message changing to 0 secs either.

File: src/js/app.js [hunk 2]
```diff
@@ -4,3 +4,7 @@ window.setLocale = function (el) {
     console.log(el.value);
     window.location.href = window.location.href.split("?")[0] + "?lang=" + el.value;
 }
+
+setInterval(function() {
+  location.reload();
+}, 60000);
```
Comment (messutied) [inline src/js/app.js]: I'd make this a one-liner, also setInterval doesn't really make sense as once its reloaded the timer its lost and its set again

```javascript
setTimeout(function() { location.reload() }, 60000);
```

**Comment by messutied** [comment]
OK then, can you add a class specific for this label though? if you look closely this class seems to be adding some margin to the right, maybe just remove the class `about`?

File: src/js/app.js (hunk 2 above)
Comment (jorge8989) [reply src/js/app.js]: true

**Pull request merged**